HelpfulItem.cpp

```diff
@@ -2,5 +2,6 @@
 
 void HelpfulItem::use(Character *character)
 {
-    
-}
\ No newline at end of file
+    useHelpfulItem(character, this);
+
+}
```

Comment:
Where is this `useHelpfulItem` function declared?
Are you `#including` the file that it is declared in? 
- [ ] revise